Dockerfile

```diff
@@ -1,37 +1,43 @@
-FROM golang:1.16-alpine AS builder
+FROM golang:1.17.2-alpine AS builder
```

Comment:
Could you explain why you made these the changes in the Dockerfile? Do they enable multiarch builds, as you mentioned in https://github.com/tus/tusd/issues/511#issuecomment-945778847?

Reply:
I was talking about all changes in the Dockerfile :) However, you second comment already explains what I wanted to know!